language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/resolvers/AbstractItemResolver.java

```diff
@@ -358,4 +365,25 @@ protected void populateBasicTypes(List<CompletionItem> completionItems, List<Sym
             }
         });
     }
+
+    /**
+     * Remove the invalid symbols such as anon types, injected packages and invokable symbols without receiver.
+     * @param symbolInfoList    Symbol info list to be filtered
+     * @return {@link List}     List of filtered symbols
+     */
+    protected List<SymbolInfo> removeInvalidStatementScopeSymbols(List<SymbolInfo> symbolInfoList) {
+        // We need to remove the functions having a receiver symbol and the bTypes of the following
+        // ballerina.coordinator, ballerina.runtime, and anonStructs
+        ArrayList<String> invalidPkgs = new ArrayList<>(Arrays.asList("ballerina.runtime",
+                "ballerina.transactions"));
```

Comment:
package name should be change to "org/pkg_name" or just "pkg_name"

Reply:
Fixed